### ISSUE TITLE
Consider PrimaryData that start before simulation

### ISSUE THREAD

**Issue by staudtMarius** [ie3-institute/simona#1034]
One point to discuss, based on the data in `vn_simona`: Asuming I would increase the simulation start time by one hour from '2011-01-01T00:00:00Z' to `2011-01-01T01:00:00Z`, this would cause the exception because time series `8c04e94e-76b0-4369-a55c-f5e1117fb83e` has no entry for 01:00:00. But with our general approach we define values as valid until a new one at a certain tick arrise. In my humble opinion this case would be vaild, while some earlier start date (e.g. 2010-12-31T23:00:00Z') wouldn't.

_Originally posted by @danielfeismann in https://github.com/ie3-institute/simona/issues/1016#issuecomment-2473095811_

**Comment by sebastian-peter** [comment]
I agree. Also linking the post with my thoughts: https://github.com/ie3-institute/simona/pull/1016#pullrequestreview-2444731903
